src/test/java/automation/tests/ManagingGroupTest.java

```diff
@@ -0,0 +1,75 @@
+package automation.tests;
+
+import automation.utils.loaders.EnvironmentConfigLoader;
+import automation.utils.loaders.GroupPropLoader;
+import automation.utils.loaders.UserConfigLoader;
+import automation.pages.BrowsePage;
+import automation.pages.LoginPage;
+import automation.pages.NewGroupPage;
+import org.junit.jupiter.api.*;
+import org.openqa.selenium.WebDriver;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class ManagingGroupTest {
+
+    private static LoginPage loginPage;
+    private static WebDriver driver;
+    private static UserConfigLoader userConfLoader = new UserConfigLoader("user");
+    private static EnvironmentConfigLoader envConfLoader = new EnvironmentConfigLoader("environment");
+    private NewGroupPage newGroupPage;
+    private BrowsePage browsePage;
+    private GroupPropLoader groupPropLoader = new GroupPropLoader("group");
+
+    //TC01 - New group can be added
+    @Test
+    void checkIfNewGroupCanBeCreated() {
+        //given:
+        driver.get(envConfLoader.getNewGroupPanel());
+
+        //when:
+        String identifier = groupPropLoader.getGroupIdentifier();
+        String displayName = groupPropLoader.getGroupDisplayName();
+
+        newGroupPage = new NewGroupPage(driver);
+        newGroupPage.typeIdentifier(identifier);
+        newGroupPage.typeDisplayName(displayName);
+        newGroupPage.submitCreateGroupButton();
+
+        browsePage = new BrowsePage(driver, displayName, identifier);
+
+        driver.get(envConfLoader.getAdminBrowsePanel());
```

Comment:
Do not repeat yourself. You call driver.get(envConfLoader.<medhod>) about 20 times.
Create an Enum with all pages you have in application and extend it with a class named e.g. Go. So you can navigate using Go.To(Pages.ADMIN_PANEL); This is more readable. Don't forget about naming pages based on their business value, not their URL.

Reply:
More or less done. I write an enum and pass Pages via go.to(Pages.PAGE) but have to make it move addresses from enum to make it more DRY.